concent_api/core/message_handlers.py

```diff
@@ -1,33 +1,37 @@
-import datetime
-
 import copy
-from base64                         import b64encode
-from django.conf                    import settings
-from django.http                    import HttpResponse
-from django.utils                   import timezone
-from golem_messages                 import message
-from golem_messages.datastructures  import MessageHeader
-
-from core.exceptions                import Http400
-from core.models                    import Client
-from core.models                    import PaymentInfo
-from core.models                    import PendingResponse
-from core.models                    import StoredMessage
-from core.models                    import Subtask
-from core.payments                  import base
-from core.validation                import validate_golem_message_reject
-from core.validation                import validate_golem_message_task_to_compute
-from core.validation                import validate_id_value
-from core.validation                import validate_report_computed_task_time_window
-from core.subtask_helpers           import verify_message_subtask_results_accepted
-from core.transfer_operations       import store_pending_message
-from core.transfer_operations       import create_file_transfer_token
-from utils                          import logging
-from utils.helpers                  import decode_client_public_key
-from utils.helpers                  import decode_other_party_public_key
-from utils.helpers                  import deserialize_message
-from utils.helpers                  import get_current_utc_timestamp
-from utils.helpers                  import parse_timestamp_to_utc_datetime
+import datetime
```

Comment:
Didn't we agreed to keep this formatting for imports ?

Reply:
Ok, I just thought we will keep them for imports.

Reply:
I would also be for keeping it for imports but unfortutely I don't think PyCharm can handle that.